src/bin/wake.rs

```diff
@@ -0,0 +1,28 @@
+use clap::Parser;
+
+#[derive(Parser)]
+#[clap(about = "WakeOnLan a device. https://github.com/LesnyRumcajs/wakey.git", long_about = None)]
+struct CmdArgs {
+    /// mac address to send packet to
+    #[clap(short, long)]
+    mac: Option<String>,
+}
+fn main() {
+    let args = CmdArgs::parse();
+    if let Some(m) = args.mac {
+        let mut sep: char = '/';
```

Comment:
I believe you could shorten it to something along the lines of `let sep = m.chars().find(|ch| ch == ':' || ch == '-').unwrap_or('/');`. What do you think?

Reply:
Yes! sure. To be honest, this is not the best solution.
Didn't know about the wolwaker... :-)